packages/botonic-cli/src/botonic-playground-service.ts

```diff
@@ -6,7 +6,19 @@ import { PlaygroundSessionInfo } from './interfaces'
 import { getCurrentDirectory } from './util/file-system'
 
 const PLAYGROUND_API_HOST = 'https://api.playground.botonic.io'
-
+/**
+ * Temporary solution to allow aborting promise after a certain timeout.
+ */
+function withTimeout<T>(
+  promise: Promise<T>,
+  errorMsg: string,
+  timeout = 2 * 1000
```

Comment:
Specify the units in the variable's name -> `timeoutMs`?

Reply:
changed! 👍